float32[] left_lane
float32[] right_lane
float32 lateral_deviation
float32 yaw
